### DISCUSSION TITLE
(Feature Requests) Changing the announcer lines.

### DISCUSSION THREAD

**Discussion by makoflagkk** [TeamRizu/OutFox#687] [03 - Feature Requests]
I would I like to see if the announcer was recoded to replicate the same behavior shown in DDR Ace based games such as DDR A, A20, A20 PLUS, A3 and even DDR GRAND PRIX. If this can't be done due to a copyright issue let me know. But if possible I would like to see this behavior implemented in future builds of Outfox. The thing I don't like about how the StepMania announcer system works is that the announcer does not count combos all the time. They do it like 60 percent of the time. It reacts in a inappropriate way to certain events. For example if you get a new record and fail and song the announcer will the announcer will still congratulate you too. Even if you failed. It also seems very delayed and choppy the way the announcer fires lines. It does not contain a cheering flag. I will also like to see a future universal compatibly with all themes. Because on certain themes I use, the announcer fires off the wrong line varying from theme to theme. Thank you

**Comment by ghost** [comment]
Maybe check your audio files to make sure they ain't causing the acting up.
(btw if they already fixed this or it's been too long, ok)

**Reply by makoflagkk** [reply]
> Maybe check your audio files to make sure they ain't causing the acting up.
> 
> (btw if they already fixed this or it's been too long, ok)

I don't want to be rude but why do you keep commenting on my issues on the Outfox repo. The dev team wants nothing to do with me anymore. So please don't comment any more under my GitHub profile because they have linked to their discord server if I remember correctly and they don't want to see my name there. I am trying to be on good terms with them so don't make them angry. Sorry if this sounds rude man. I don't mean  this on the repo. I know there is another tab for off topic conversations so if you wanna talk on there go ahead. This is really revenant to.

**Reply by ghost** [reply]
Ah, I just comment around on different places. Don't really take who's talking into account. But I understand. I'll try to stay  away from your comments.
